ovs/extensions/healthcheck/alba/alba_health_check.py

```diff
@@ -306,6 +306,8 @@ def _get_all_responding_backends(result_handler):
             for stack in alba_backend.local_stack.values():
                 for osds in stack.values():
                     for asd in osds['asds'].values():
+                        if 'alba_backend_guid' not in asd:
+                            continue
                         if alba_backend.guid != asd['alba_backend_guid']:
```

Comment:
you also do this:

if alba_backend.guid != asd.get('alba_backend_guid', 'unavailable')

that would make the 2 lines above unnecessary ...